docs/statistics/cuped.md

```diff
@@ -6,17 +6,53 @@ However, there are options beyond waiting to gather more data, and all these opt
 
 ![CUPED variance reduction explained](/img/measuring-experiments/cuped-variance-plot.png)
 
-An illustration of how CUPED helps reduce the variance in an effect estimate, leading to a much tighter confidence interval.
+The above sketch illustrates how CUPED helps reduce the variance in an effect estimate, leading to a much tighter confidence interval.
 
-## Overview
+:::info
 
-When CUPED is enabled, Eppo automatically ingests aggregated data about each subjects' events across metrics that are added to your experiment for the 30 days before they joined the experiment. Eppo then uses a ridge regression model to predict subject-level outcomes for each metric. For each metric, we predict outcomes using every available metric. These predictions are then used to get more accurate effect estimates, reducing the variance and hence leading to better confidence intervals.
+Note that we only use CUPED to reduce variance on the metrics overview page.
+In particular, not that there are no CUPED estimates for:
```

Comment:
> In particular, not that there are no CUPED estimates for:

This should be "note"

Can we add an explanation as to why we only focus on the metrics overview page?